home/models.py

```diff
@@ -41,24 +62,49 @@ def render_basic(self, value, context=None):
         pass
 
 
-def get_choices():
+def get_valid_profile_values(field):
     if SiteSettings.objects.first():
-        return [
-            (choice, choice)
-            for choice in SiteSettings.objects.first().content_variations_options
-        ]
+        profile_values = {}
+        for profile_block in SiteSettings.objects.first().profile_field_options:
+            profile_values[profile_block.block_type] = [b for b in profile_block.value]
+        return profile_values[field]
     return []
 
 
+def get_gender_choices():
+    # Wrapper for get_profile_field_choices that can be passed as a callable
+    choices = {k: v for k, v in GENDER_CHOICES}
```

Comment:
what is k and v?

Reply:
They are variables for the key and value. They are being used for the list comprehension so they don't exist outside of this line

Reply:
👍